crates/api_models/src/payments.rs

```diff
@@ -5276,8 +5282,8 @@ pub struct RetrievePaymentLinkResponse {
     pub payment_link_id: String,
     /// Identifier for Merchant
     pub merchant_id: String,
-    /// Payment Link
-    pub link_to_pay: String,
+    /// Open payment link (without any security checks and listing SPMs)
+    pub link_open: String,
```

Comment:
Is it alright to update this API response @sahkal ?

Reply:
no, i think only `link`should suffice for open link and should the `link_secure` be called `secure_link`